xl-objects.go

```diff
@@ -138,11 +137,32 @@ func (xl xlObjects) GetObject(bucket, object string, startOffset int64) (io.Read
 	return fileReader, nil
 }
 
+// Return the parts of a multipart upload.
+func (xl xlObjects) getParts(bucket, object string) (parts completedParts, err error) {
```

Comment:
Make this function stateless - pass "xl.storage" as part of the argument itself. 

"getParts(xl.storage, bucket, object)"  - since there is no other need for passing xl.storage inside.